ophyd/tests/test_redis_signal.py

```diff
@@ -0,0 +1,42 @@
+import time
+import fakeredis
```

Comment:
If you add `fakeredis` to https://github.com/bluesky/ophyd/blob/master/test-requirements.txt it will get automatically installed on travis.